app/graphql/queries/fetch_places.rb

```diff
@@ -5,12 +5,13 @@ class FetchPlaces < Queries::BaseQuery
     argument :city, String, required: true
     argument :country, String, required: true
     argument :categories, [String], required: false, default_value: nil
-    argument :page, Integer, required: false, default_value: 0
-    argument :radius, Integer, required: false, default_value: 2500
+    argument :page_token, String, required: false, default_value: nil
+    argument :search_type, String, required: false, default_value: "google"
+    argument :search_radius, Integer, required: false, default_value: 2500
 
     def resolve(args)
-      city_info = GeocodingFacade.city_info(args[:city], args[:country])
-      places = PlacesFacade.places(city_info, args[:categories], args[:page], args[:radius])
+      city_info = GeocodingFacade.city_info(args.delete(:city), args.delete(:country))
```

Comment:
Pretty curious what the .delete is for over here if you'd be willing to briefly explain!